server/index.js

```diff
@@ -1,14 +1,8 @@
-const fs = require('fs');
-const https = require('https');
+const http = require('http');
 const app = require('./app');
 
-const privateKey = fs.readFileSync('./security/server.key');
-const certificate = fs.readFileSync('./security/server.crt');
 
-https.createServer({
-  key: privateKey,
-  cert: certificate,
-}, app).listen(app.get('PORT'), () => {
+http.createServer(app).listen(app.get('PORT'), () => {
```

Comment:
ummm what is this again ?

Reply:
Creating the server from docs. Could it be improved?